docs/ops/quantization/FakeQuantize_1.md

```diff
@@ -13,13 +13,14 @@
 Each element of the output is defined as the result of the following expression:
 
 ```python
-if x <= min(input_low, input_high):
-    output = output_low
-elif x > max(input_low, input_high):
-    output = output_high
+if input_low != input_high:
```

Comment:
Are these changes backward compatible? Seems like not really. Could you provide detailed explanation what has changed and wny?

Reply:
@jane-intel Previous spec behavior was not corrected for inverted intervals + values that out of bounds. To handle such values correctly for all 4 cases and not create multiple conditions (inverted/plain; in/out), I suggest just values right and pass them through a general formula.
If we are talking about branch where `input_low == input_high`, behavior is the same as on previous spec.

Reply:
Why do we need to handle inverted intervals in a special way? What does it mean that it was incorrect for out of bounds???